stellargraph/layer/graph_attention.py

```diff
@@ -615,6 +619,18 @@ class GAT:
         normalize (str or None): normalization applied to the final output features of the GAT layers stack. Default is None.
         activations (list of str): list of activations applied to each layer's output; defaults to ['relu', ..., 'relu', 'linear'].
         saliency_map_support (bool): XXX What is this?
+
+
+    Optional Args:
+        kernel_initializer (str or func): The initialiser to use for the head weights.
```

Comment:
While in the research code they exposed the intializers and regularizers for all internal options, I think we should try and guide the user and have  fewer options at the top level class.  I think having `kernel_regularizer` and `attn_kernal_regularizer` is all we need for now. The other options I think are too much, although we can still support them in the underlying layers.

This also applies to:
gcn.py: line 61 (GCN)
graphsage.py: line 732 (GraphSAGE)
graphsage.py: line 1046 (DirectedGraphSAGE)
hinsage.py: line 46 (HinSAGE)